### PR TITLE
added unit test

### PR DESCRIPTION
**Pull request by forThePurposeOfCloning**
Contributes to #47

### CODE REVIEW

**Comment by forThePurposeOfCloning** [comment]
@eytan-avisror please review

File: controllers/provisioners/ekscloudformation/ekscloudformation_test.go [hunk 1]
```diff
@@ -1272,3 +1272,14 @@ func TestGetNodeAutoScalingGroupMetrics(t *testing.T) {
 	}
 }
 
+func TestParseCustomResourceYaml(t *testing.T) {
+	empty, err := common.ParseCustomResourceYaml("")
+	if (len(empty.Object) > 0)  {
+		t.Fatal("Object should be empty")
+		t.Fail();
```
Comment (eytan-avisror) [inline controllers/provisioners/ekscloudformation/ekscloudformation_test.go]: I think `t.Fail()` is not needed here if you already do `t.Fatal()`
Also can you add a more descriptive message (for both `t.Fatal()` calls) that includes `expected: <>, got: <>`

File: controllers/provisioners/ekscloudformation/ekscloudformation_test.go [hunk 2]
```diff
@@ -1272,3 +1272,14 @@ func TestGetNodeAutoScalingGroupMetrics(t *testing.T) {
 	}
 }
 
+func TestParseCustomResourceYaml(t *testing.T) {
```
Comment (eytan-avisror) [inline controllers/provisioners/ekscloudformation/ekscloudformation_test.go]: Is the goal of this test to verify `ParseCustomResourceYaml` can handle an empty string?
Maybe we should rename the test to `TestParseCustomResourceYamlEmptyString`

File: test-bdd/main_test.go [hunk 3]
```diff
@@ -76,6 +76,8 @@ func TestE2e(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "InstanceGroup Type Suite", []Reporter{junitReporter})
 }
 
+
+
```
Comment (eytan-avisror) [inline test-bdd/main_test.go]: seems like we can undo these extra blank lines

Comment (forThePurposeOfCloning) [reply test-bdd/main_test.go]: Quick question. What files besides the .idea files would you want removed?

**Pull request merged**